data/json/items/containers.json

```diff
@@ -1783,6 +1783,93 @@
     "symbol": ")",
     "color": "red"
   },
+  {
+    "id": "mre_bag_small",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "small MRE bag" },
+    "looks_like": "mre_beef_box",
+    "description": "A small sturdy plastic container of multi-layered construction, with detailed information about its contents nutritional information.",
+    "weight": "70 g",
+    "volume": "100 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "750 ml",
+        "max_contains_weight": "2 kg",
+        "airtight": true,
+        "moves": 200,
+        "open_container": true,
```

Comment:
This is responsible for spilling item.
Butter spread and peanut butter are liquids, can't be stashed to inventory if in an open container (e.g. a cup)

Reply:
Here is what I don't understand if I just remove `open_container`, testing the `MRE` itemgroup.
![MRE_pack_no_opencontainer](https://user-images.githubusercontent.com/58050969/117880111-57d9c100-b26d-11eb-8fae-9c27dc3723e1.png)
If I keep `open_container`
![MRE_pack_opencontainer](https://user-images.githubusercontent.com/58050969/117880159-67f1a080-b26d-11eb-99be-bce90b580554.png)